cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -63,9 +65,9 @@ def new_snapshot(test_vm, has_name=True):
     return new_snapshot
 
 
-@pytest.mark.uncollectif(
-    lambda provider: provider.type != 'virtualcenter' and (provider.type != 'rhevm' or
-          (provider.type == 'rhevm' and provider.version < 4)))
+@pytest.mark.uncollectif(lambda provider:
+    (provider.type != 'virtualcenter' and provider.type != 'rhevm') or
+    (provider.type == 'rhevm' and provider.version < 4))
```

Comment:
It's discouraged to use ``provider.type``, use ``provider.one_of(RHEVMProvider)`` instead. So here it would be
``not provider.one_of(VMwareProvider, RHEVMProvider) or (provider.one_of(RHEVMProvider) and provider.version < 4))``
And similar at other places where you use ``provider.type``. Sorry I haven't noticed this right away.